docs/zh/UserGuide/Administration-Management/Administration.md

```diff
@@ -113,6 +113,27 @@ IoTDB> INSERT INTO root.ln.wf01.wt01(timestamp, status) values(1509465600000, tr
 Msg: The statement is executed successfully.
 ```
 
+### 撤销用户权限
+
+授予用戶權限後，我們可以使用 `REVOKE USER <userName> PRIVILEGES <privileges> ON <nodeName>` 來撤銷已授予的用戶權限。 例如：
```

Comment:
Should we use the simplified Chinese here to keep the consistency. :)

Reply:
Oh, sorry for this. I'll correct it.